docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+# WARNING!
+# This is a development version of the docker-compose.yml file.
+# Avoid using this file in your production environment.
+# We're exposing here sensitive ports and mounting code volumes for rapid development and debugging of the server stack.
+
+x-logging: &x-logging
+  logging:
+    driver: 'json-file'
+    options:
+      max-file: '5'
+      max-size: '10m'
+
+version: '3'
+
+services:
+  openruntimes-executor:
+    container_name: openruntimes-executor
+    hostname: exc1
+    <<: *x-logging
+    stop_signal: SIGINT
+    build:
+      context: .
+    networks:
+      runtimes:
+    ports:
+      - 9900:80
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./app:/usr/local/app:rw
+      - ./src:/usr/local/src:rw
+      - openruntimes-builds:/storage/builds:rw
+      - openruntimes-functions:/storage/functions:rw
+      - /tmp:/tmp:rw
+      - ./tests/resources/functions:/storage/functions:rw
+    environment:
+      - OPEN_RUNTIMES_EXECUTOR_ENV
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_DEVICE
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_ACCESS_KEY=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_SECRET=
+      - OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_REGION
```

Comment:
I think there is a difference between having an `=` after the env variable name and not having it 

Having an `=` after the env variable essentially means we are equating it to an empty string ""
Not having an `=` after the variable indicates that it will be supplied through the environment.

Since we are supplying all vars from an external env file, we should remove `=` from all the vars

Reply:
Removed all `=` from docker compose yml files.